exercise/templates/exercise/_user_results.html

```diff
@@ -16,22 +19,37 @@
 </p>
 {% endif %}
 
-<p id="toggle-expired" class="hide">
-	<button class="aplus-button--secondary aplus-button--xs">
-		<span class="glyphicon glyphicon-time" aria-hidden="true"></span>
-		{% translate "SHOW_OLDER_ASSIGNMENTS" %}
+<p>
+	<button id="toggle-expand-all-modules" class="aplus-button--secondary aplus-button--xs" style="display:none">
+		<span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+		{% translate "EXPAND_ALL_MODULES" %}
 	</button>
```

Comment:
I believe the toggle button needs an aria-pressed attribute for accessibility https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#toggle_buttons.

Reply:
I don't think this is needed since the buttons have the ``aria-hidden`` class.

From Mozilla:
"Adding ``aria-hidden="true"`` to an element removes that element and all of its children from the accessibility tree"

The ``filter-categories`` buttons above don't have ``aria-pressed`` class either when toggled.

Reply:
Okay, I added this to all buttons now (even the older ones).